Directory.Packages.props

```diff
@@ -28,6 +28,7 @@
     <PackageVersion Include="Marten" Version="7.26.1" />
     <PackageVersion Include="MathNet.Numerics" Version="5.0.0" />
     <PackageVersion Include="MediatR" Version="12.2.0" />
+    <PackageVersion Include="MessagePack" Version="2.5.108" />
```

Comment:
Why the ancient version?

Reply:
Fixed. We are now using the latest version, `2.5.172`.
I believe `MassTransit.SignalR` has a transitive dependency on the previous version, but using a later version shouldn't cause any problems.